spec/features/view_event_spec.rb

```diff
@@ -64,12 +68,14 @@
       let(:member) { Fabricate(:member) }
 
       before do
-        login(member)
+        login_mock_omniauth(member, 'Log in')
       end
 
       context 'can RSVP to an event' do
 
         scenario "as a Coach" do
+          expect(current_path).to eq(event_path(open_event))
```

Comment:
I added this line to the tests that require login to verify the redirect is working.